src/ai/backend/agent/kubernetes/kernel.py

```diff
@@ -202,6 +202,43 @@ async def get_service_apps(self):
         result = await self.runner.feed_service_apps()
         return result
 
+    async def check_commit_tag(self, commit_path, get_lock):
+        log.error("Committing in Kubernetes is not supported yet.")
+
+    async def commit(self, image_commit_path, path):
+        log.error("Committing in Kubernetes is not supported yet.")
+        # assert self.runner is not None
```

Comment:
Add `TODO:` or `FIXME:` comment so that future developers could refer there is something to do here.
Also, it's better to explicitly raise `NotImplementedError` instead of just logging it and doing nothing, particularly when the function is expected to return some values.